modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/StickerTag.java

```diff
@@ -209,6 +218,12 @@ protected int processStartTag() throws Exception {
 
 		jspWriter.write("\">");
 
+		if (Validator.isNotNull(_content)) {
```

Comment:
This won't work as expected in a `jsp` which should be the case for something called "content" or at least in spirit. I don't think we want to pass string content.

When I said "add it if it makes sense" I meant adding explicitly `_image` just like we have `_icon`. 

I think that if we want to implement arbitrary content, we should do so following jsp tag conventions and not an additional attribute.

Reply:
Actually thanks to this comments I realize that neither in Clay v2 or v3 we were supporting `stickerLabel` to pass a text for the sticker, which Clay defines as an option [here](https://clayui.com/docs/components/sticker.html#user-icon)

So you're right, I'm changing this to accept `imageSrc` instead content, which will result in having in `StickerTag` those three options: `icon`, `imageSrc` and `label`. I'm also opening an issue in Clay so we support sticker label in `CardWithUser` component.

Reply:
https://github.com/liferay/clay/issues/3763